### PR TITLE
Flux sample package

### LINKED ISSUE #2845: Update sample data content
**PR:** https://github.com/influxdata/flux/pull/3873

The next release of Flux includes a new `influxdata/influxdb/sample` package which provides a quick way to download and use our sample datasets. We need to:

- [x] Document the `influxdata/influxdb/sample` package
- [x] Update the OSS and Cloud sample data pages to use the updated sample data sets.

##### Relevant URLs
- https://docs.influxdata.com/influxdb/cloud/reference/sample-data/
- https://docs.influxdata.com/influxdb/v2.0/reference/sample-data/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #2845

Adds the new Flux `influxdata/influxdb/sample` package and updates sample data instructions to use the `sample.data()` function to load certain data sets.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud/reference/sample-data.md [hunk 1]
```diff
@@ -24,46 +24,117 @@ Demo data is not available for use with third-party integrations such as Grafana
...
-This data set is  publicly available data from the [National Oceanic and Atmospheric Administration’s (NOAA) Center for Operational Oceanographic Products and Services](http://tidesandcurrents.noaa.gov/stations.html).
+{{% caption %}}
+**Size**: ~1.3 MB • **Updated**: every 15m
+{{% /caption %}}
 
-[The CSV data](https://influx-testdata.s3.amazonaws.com/noaa.csv) includes 15,258
-observations of water levels (ft) collected every six minutes at two stations
+The **NOAA National Data Buoy Center (NDBC)** dataset provides the latest
+observations from the NOAA NDBC network of buoys throughout the world.
+Observations are updated approximately every 15 minutes.
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/sample-data.md:100]:
```suggestion
The **NOAA National Data Buoy Center (NDBC)** dataset provides observations (updated every 15 minutes) from the NOAA NDBC network of buoys throughout the world.
```

File: content/influxdb/cloud/reference/sample-data.md [hunk 2]
```diff
@@ -24,46 +24,117 @@ Demo data is not available for use with third-party integrations such as Grafana
...
+
+To download and output the most recent NOAA NDBC observations, use the
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/sample-data.md:102]: is most recent relative to any measure?

File: content/influxdb/cloud/reference/flux/stdlib/influxdb-sample/_index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: Flux InfluxDB sample package
+list_title: InfluxDB sample package
+description: >
+  The Flux InfluxDB sample package provides functions for downloading and outputting InfluxDB data sample.
+  Import the `influxdata/influxdb/sample` package.
```
Comment (mluu12) [inline content/influxdb/cloud/reference/flux/stdlib/influxdb-sample/_index.md:6]: Hi Scott! I was just wondering if the sample data would later require a full procedural doc?

File: content/influxdb/cloud/reference/flux/stdlib/influxdb-sample/_index.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+menu:
+  influxdb_cloud_ref:
+    name: InfluxDB Sample
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/influxdb/cloud/reference/flux/stdlib/influxdb-sample/_index.md:9]: Updated all of these

File: content/influxdb/cloud/reference/flux/stdlib/influxdb-sample/_index.md (hunk 3 above)
Comment (kelseiv) [reply content/influxdb/cloud/reference/flux/stdlib/influxdb-sample/_index.md:6]: @mluu12 @sanderson a procedure would fit nicely in with "Get started with InfluxDB"...maybe "Explore sample data in UI"? 
While we do use sample data in many procedures, right now, if folks search for "sample data" and land on the "Sample data" page, there's no instruction for how to use/explore sample data in the product.

**Pull request merged**